runtime/jit_vm/ctsupport.c

```diff
@@ -276,22 +277,24 @@ jitCTResolveInstanceFieldRefWithMethod (J9VMThread *vmStruct, J9Method *method,
 	if (0 != resolveFlags) {
 		resolveFlagsBuffer |= J9_RESOLVE_FLAG_FIELD_SETTER;
 	}
-	result = vmStruct->javaVM->internalVMFunctions->resolveInstanceFieldRefInto(
+	result = vm->internalVMFunctions->resolveInstanceFieldRefInto(
 			vmStruct, method, constantPool, fieldIndex, resolveFlagsBuffer, &resolvedFieldBuffer, NULL);
 
 	if (-1 != result) {
 		*resolvedField = resolvedFieldBuffer;
 	}
-#if defined (J9VM_JIT_FULL_SPEED_DEBUG)
-	ALWAYS_TRIGGER_J9HOOK_JIT_CHECK_FOR_DATA_BREAKPOINT(vmStruct->javaVM->jitConfig->hookInterface,
-			vmStruct,
-			result,	/* can be modified */
-			fieldIndex,
-			constantPool,
-			*resolvedField,
-			0,
-			resolveFlags);
-#endif
+
+	if (J9_ARE_NO_BITS_SET(vm->extendedRuntimeFlags, J9_EXTENDED_RUNTIME_JIT_INLINE_WATCHES)) {
```

Comment:
This probably needs a comment to explain why compile time resolves don't need to check for watches anymore (in this mode)

Reply:
Done.